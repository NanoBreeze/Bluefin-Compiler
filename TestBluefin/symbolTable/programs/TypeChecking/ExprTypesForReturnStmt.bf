int f() {
	return 45;
}

int g() {
	int a = 9;
	return a;
}

float h() {
	int a = 1;
	return a;
}

struct First {
	int one;
};

float i() {
	First first;
	return first.one + 9;
}