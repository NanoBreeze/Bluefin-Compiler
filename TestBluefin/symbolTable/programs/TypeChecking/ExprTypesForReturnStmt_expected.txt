45 - t_int - t_int
9 - t_int - t_int
a - t_int - t_int
1 - t_int - t_int
a - t_int - t_float
first - t_First - t_First
first.one - t_int - t_int
9 - t_int - t_int
first.one+9 - t_int - t_float
